app/validators/edu_program_validator.rb

```diff
@@ -5,7 +5,8 @@ def validate(record)
     duplicates_exist = EduProgram.where(['vet_tec_program = ? AND facility_code = ? AND id != ?',
                                          record.vet_tec_program, record.facility_code, record.id]).any?
     record.errors[:base] << non_unique_error_msg(record) if duplicates_exist
-
+    record.errors[:base] << 'The Facility Code is blank:' if record.facility_code.blank?
+    record.errors[:base] << 'The VET TEC Program (Program Name) is blank:' if record.vet_tec_program.blank?
```

Comment:
Can we get some specs in `edu_program_validator_spec` covering these cases?

Reply:
added